tests/storage/cdi_import/test_import_http.py

```diff
@@ -684,15 +684,15 @@ def test_vm_from_dv_on_different_node(
             {
                 "dv_name": "dv-win-19",
                 "source": HTTP,
-                "image": f"{Images.Windows.UEFI_WIN_DIR}/{Images.Windows.WIN19_RAW}",
```

Comment:
We want to have at least one test that imports a very large image, like WIN19_RAW. Can we keep it in this test?